### ISSUE TITLE
Dead project ?

### ISSUE THREAD

**Issue by humbienri** [datakurre/robotframework-selenium2screenshots#18]
Hello,

Is this project dead?  Looks rather interesting!  I sort of cheated off of you as to how to inject jQuery into RF-SL.  But it seems that it has to be done on a page by page basis?

I was hoping some of your documentation had included "how to use it" examples.  That's how I noticed that it seemed rather dormant, the project.

**Comment by datakurre** [comment]
Yes, has to be ”bootstrapped” on every page. This was written in time when browsers had less native selector functions than today.

Docs have some examples https://robotframework-selenium2screenshots.readthedocs.io/en/latest/

This project is in “maintenance mode”, maintained by people who still rely on Selenium2Library before it was named to SeleniumLibrary.

I did a refactoring last year, but with some breaking changes in keyword options https://pypi.org/project/robotframework-seleniumscreenshots/
